imgparse/imgparse.py

```diff
@@ -142,64 +129,62 @@ def get_timestamp(image_path=None, exif_data=None, format_string="%Y:%m:%d %H:%M
             exif_data["EXIF DateTimeOriginal"].values, format_string
         )
     except KeyError:
-        logger.error("Couldn't determine image timestamp.")
-        raise ParsingError("Couldn't determine image timestamp.")
+        logger.error("Couldn't parse image timestamp. Sensor might not be supported")
+        raise ParsingError(
+            "Couldn't parse image timestamp. Sensor might not be supported"
+        )
     except ValueError:
-        logger.error("Couldn't parse found timestamp with given format string.")
-        raise ValueError("Couldn't parse found timestamp with given format string.")
+        logger.error("Couldn't parse found timestamp with given format string")
+        raise ParsingError("Couldn't parse found timestamp with given format string")
 
-    make, model = get_make_and_model(image_path=image_path, exif_data=exif_data)
+    make, model = get_make_and_model(image_path, exif_data)
     if make == "Sentera":
         datetime_obj = pytz.utc.localize(datetime_obj)
-    elif make == "DJI":
-        lat, lon = get_lat_lon(image_path=image_path, exif_data=exif_data)
+    else:
+        lat, lon = get_lat_lon(image_path, exif_data)
         timezone = pytz.timezone(TimezoneFinder().timezone_at(lng=lon, lat=lat))
         datetime_obj = timezone.localize(datetime_obj)
-    else:
```

Comment:
Why is it safe to remove this case?

Reply:
We can localize datetime by lat/lon for all non-Sentera sensor makes, not just DJI

Reply:
See here: https://github.com/SenteraLLC/py-image-metadata-parser/pull/49/files/29b146deb076676d51d00717d07dcda4041607bc#diff-69bd3a008c817f4c9feda3865ab8473e91a99e6c6026f908531c4eb38e35ad62R143